### ISSUE TITLE
Pre training file configuration issue

### ISSUE THREAD

**Issue by JunsFu** [thuiar/Self-MM#24]
Model name 'F:\MLP\2023_Summer\Git\Self-MM\pretrained_model\bert-base-uncased' not found in model shortcut name list (bert-base-uncased, bert-large-uncased, bert-base-cased, bert-large-cased, bert-base-multilingual-uncased, bert-base-multilingual-cased, bert-base-chinese, bert-base-german-cased, bert-large-uncased-whole-word-masking, bert-large-cased-whole-word-masking, bert-large-uncased-whole-word-masking-finetuned-squad, bert-large-cased-whole-word-masking-finetuned-squad, bert-base-cased-finetuned-mrpc, bert-base-german-dbmdz-cased, bert-base-german-dbmdz-uncased, bert-base-finnish-cased-v1, bert-base-finnish-uncased-v1, bert-base-dutch-cased). Assuming 'F:\MLP\2023_Summer\Git\Self-MM\pretrained_model\bert-base-uncased' is a path, a model identifier, or url to a directory containing tokenizer files.
Didn't find file F:\MLP\2023_Summer\Git\Self-MM\pretrained_model\bert-base-uncased\added_tokens.json. We won't load it.
Didn't find file F:\MLP\2023_Summer\Git\Self-MM\pretrained_model\bert-base-uncased\special_tokens_map.json. We won't load it.
Didn't find file F:\MLP\2023_Summer\Git\Self-MM\pretrained_model\bert-base-uncased\tokenizer_config.json. We won't load it.
loading file F:\MLP\2023_Summer\Git\Self-MM\pretrained_model\bert-base-uncased\vocab.txt
![1696862325181](https://github.com/thuiar/Self-MM/assets/100461491/2a2668c8-fc16-4c41-a502-299559f85fe9)

May I ask the correct method for placing pre training files? Thank you very much！！

**Comment by JunsFu** [comment]
![image](https://github.com/thuiar/Self-MM/assets/100461491/ca707e56-72f4-4507-9b81-02ccbdd643db)
For the convenience of observing errors, I have placed a separate image

**Comment by YangDargon** [comment]
Is it solved？

**Comment by JunsFu** [comment]
Yes，you can visit the followling link to find the&nbsp;&nbsp;Pretraining File .MMSA - Google 云端硬盘





***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "thuiar/Self-MM"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月26日(星期五) 上午9:49
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [thuiar/Self-MM] Pre training file configuration issue (Issue #24)





 
Is it solved？

**Comment by YangDargon** [comment]
Can you be more specific?

**Comment by zhaoyuanyi** [comment]
> Can you be more specific?

Is it solved？
